docs/index.md

```diff
@@ -0,0 +1,10 @@
+# A Low-Code Real-Time Kafka Stream Processor
+
+Kahpp is a self-service Kafka Stream processor capable of filtering, transforming and routing Kafka records in real-time with just a YAML file.  
```

Comment:
```suggestion
Kahpp is a self-service Kafka Stream processor capable of filtering, transforming and routing Kafka records in real-time with a YAML file.
```

Words like `just` and `simple` aren't something that I would use - there's more than just a YAML file to be done to have KaHPP up and running, unfortunately.